src/m365/spo/commands/list/list-roleinheritance-break.ts

```diff
@@ -70,6 +75,15 @@ class SpoListRoleInheritanceBreakCommand extends SpoCommand {
           return isValidSharePointUrl;
         }
 
+        const listOptions: any[] = [args.options.listId, args.options.listTitle, args.options.listUrl];
```

Comment:
We shouldn't do this manually. Instead, we should use optionSets with the three options, like:

```ts
['listId', 'listTitle', 'listUrl']
```

Let's remove lines 78-86 in favor of an option set